Labs/Alyssa_Giedd_Lab2.ipynb

```diff
@@ -0,0 +1,477 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x22710e71b38>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#B) If you have to look for a signal in 10k pixels, what is the probability distribution of your background? \n",
+    "#(Clearly state the statistical question, then turn that into math using your background distribution from part 1 of this problem.)\n",
+    "print(\"What is the likelihood that the background gave me a signal as signal-like or more than the observed value?\")\n",
+    "plt.plot(x, bg*10000)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 83,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Sigma value for 10k pixels at 18 = 4.267904750513104\n"
+     ]
+    }
+   ],
+   "source": [
+    "#C) Taking your brightest candidate signal from the region (assume it has the same signal as in part 1), \n",
+    "#calculate the significance of your detection.\n",
+    "probability = (1 - stats.norm.cdf(18, loc = 0, scale = 3))*10000\n",
+    "sigma = stats.norm.ppf(1 - probability)\n",
+    "print(\"Sigma value for 10k pixels at 18 =\", sigma)"
```

Comment:
is this sigma value significant enough for a discovery? i think you should include that here